testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -821,3 +821,6 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I do not see "Loading" text
     Then I should not see a "Operation not successful" text
     And I should not see a warning nor an error sign
+
+  Scenario: Detect repository synchronization issues in reposync logs
+    Then the reposync logs should not report errors
```

Comment:
```suggestion
  @susemanager
  Scenario: Detect repository synchronization issues in reposync logs
    Then the reposync logs should not report errors
```
About having this on BV, I think we should distinguish Uyuni than Susemanager, the reason behind it is that for Uyuni BV that will run periodically and unattended I think is better if we don't stop the pipeline due to issues at that point, as sometimes it can recover itself or we can catch other important issues.

Reply:
I disagree, I'd rather detect issues in Uyuni as well.

Especially that we tend to detect issues early when we release Uyuni.